js/scripts/dimension/playground_dimension.js

```diff
@@ -92,9 +92,10 @@ const loadPlaygroundDimension = () => {
         GAME.addEntity(new DialogBubble(CHAD, "Hey pal! My name's Papa Chad", DialogBubble.NORMAL));
         GAME.addEntity(new Crosshair());
         GAME.addEntity(new Slingshot());
- 
-        CHAD.x = -3 * Block.SCALED_SIZE;
-        CHAD.y = -20 * Block.SCALED_SIZE;
+
+        // CHAD.x = -3 * Block.SCALED_SIZE;
```

Comment:
Do we need these comments here? @NathanHinthorne

Reply:
We definitely don't. That was an accident.